Part A - Basics/08_functions/examples/func_lambda_syntax.py

```diff
@@ -3,19 +3,19 @@
 """
 lambda [param1, param2, ..]: expression
 
-Lambda functions are one-line functions which return an expression using the pre-defined
-parameters param1, param2, ... paramN.
+Lambda functions are one-line functions which return an expression using
+the pre-defined parameters param1, param2, ... paramN.
 
-Lambda functions are normally used for quick data operations on data, most notably in combination
-with map, filter, reduce.
+Lambda functions are normally used for quick operations on data,
+most notably in combination with map, filter, reduce.
 
 """
 
 # Define a list to iterate over
 data_in = [1, 2, 3]
 
 # Use a lambda function to square the input and then map the result to a list `data_out`
-data_out = map(lambda x: x * x, data_in)
+data_out = list(map(lambda x: x * x, data_in))
 
 # Print the result
-print(data_in, data_out)
```

Comment:
**Note**

Very good, this is a typical example of a difference in the behavior of Python 2/3. Please add a comment clarifying this compatibility issue.